showpass/Showpass-E2E-tests
# Sign-Up modal window element appearance test

**Description:**
Create a test script to verify the Sign Up (Create Account) modal window overall appearance. The following elements are to be verified:
- ‘Showpass’ logo
- ‘Create Account’ header
- ‘Already have an account?’ text
- ‘Log In’ link
- ‘First Name’ label and input field
- ‘Last Name’ label and input field
- ‘Email Address’ label and input field
- ‘Password’ label and input field
- ‘Confirm Password’ label and input field
- ‘Create Account’ button
- ‘Sign up using Facebook’ button'
- ‘By creating an account you agree to our’ text
- ‘privacy policy’ link
- ‘terms of service’ link

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-2

**How to test:**
Run the following test script and ensure it passes
cypress/e2e/registration/signup-modal-window-appearance.cy.js

**Test execution results:**
<img width="1717" alt="signup-modal-window-appearance cy_1" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/0adf2386-347b-404c-9a72-678e04cf6dff">
<img width="1043" alt="signup-modal-window-appearance cy_2" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/bf8aa027-cf18-494a-b37d-a064403ce48f">
